components/CheckoutForm.tsx

```diff
@@ -8,7 +8,20 @@ import {
 import { Button } from "@cheapreats/react-ui";
 import styled from "styled-components";
 
-export const CheckoutForm = (): React.ReactElement => {
+interface CheckoutFormProps {
+  orderId: string;
+}
+
+declare var process: {
+  env: {
+    BASE_URL: string;
+  };
+};
+
```

Comment:
It will be very tedious for you to declare this everywhere you use process. Have an index.d.TS file

D stands for definitions